src/new-client/src/App/Trades/TradesHeader/TradesHeader.tsx

```diff
@@ -30,17 +47,49 @@ const HeaderToolbar = styled("div")`
   align-items: center;
   justify-content: flex-end;
 `
-
-export const TradesHeader: React.FC = () => (
-  <TradesHeaderStyle>
-    <HeaderLeftGroup>Trades</HeaderLeftGroup>
-    <HeaderRightGroup>
-      <ExcelButton />
-      <HeaderToolbar>
-        <AppliedFilters />
-        <QuickFilter />
-      </HeaderToolbar>
-      <Fill />
-    </HeaderRightGroup>
-  </TradesHeaderStyle>
-)
+var isTornOut = false
+export const TradesHeader: React.FC = () => {
+  const tearOutTileState = useTearOutTileState("Tiles")
+  const tearOutAnalyticsState = useTearOutAnalyticsState("Analytics")
+  useObservableSubscription(
+    tearOutEntryTrades$.subscribe(async ([tornOut]) => {
+      if (tornOut && !isTornOut) {
+        isTornOut = true
+        openWindow(
+          {
+            url: constructUrl("/trades"),
+            width: window.innerWidth * 0.85,
+            name: "",
+            height: window.innerHeight * 0.25,
+          },
+          () => {
+            isTornOut = false
+            tearOutTrades(false)
+          },
+        )
+      }
+    }),
+  )
+  return (
+    <TradesHeaderStyle>
+      <HeaderLeftGroup>Trades</HeaderLeftGroup>
+      <HeaderRightGroup>
+        <ExcelButton />
+        <HeaderToolbar>
+          <AppliedFilters />
+          <QuickFilter />
+        </HeaderToolbar>
+        {!isTornOut && !(tearOutTileState && tearOutAnalyticsState) && (
```

Comment:
We need to be able to not show this when in OpenFin (as it handles popout layout automatically).
Options include - 
1. Adding an optional prop to the parent `Trades` component where web can say 'I support tear outs' and openfin says 'I don't support tear outs'
2. Create a file that determines if the component supports tear out and let the vite build handle the import replacement

Number 2 would look something like
```
// /App/Trades/supportsTearOut.ts 
export const supportsTearOut = true
```

```
// /App/Trades/supportsTearOut.openfin.ts 
export const supportsTearOut = false
```

```
import { supportsTearout } from './supportsTearOut'
// supportsTearout will be true when compiled for web, false when compiled for openfin
```